joomla/joomla-cms
# [4.0] Add markup to move buttons to next line

### Summary of Changes
Add markup to move buttons to next line.


### Testing Instructions
Go to Global Configuration > Installer
Change `Updates Caching (in hours)` to 0.
Go to Post Installation Messages.
See other messages where buttons are below messages.

![update-notification](https://user-images.githubusercontent.com/368084/83434397-421e1380-a3ef-11ea-878b-965849ff6487.png)
